src/extension.ts

```diff
@@ -558,6 +558,15 @@ async function getSopsGeneralOptions() {
 		sopsGeneralEnvVars[AGE_KEY_FILE_ENV_VAR_NAME] = ageKeyFile;
 	}
 
+	// Needed to work with sops via aqua (https://github.com/aquaproj/aqua).
+	// Error related to this issue https://github.com/getsops/sops/issues/884 is thrown otherwise.
+	sopsGeneralArgs.push('--config', '/dev/null');
+	// Moreover sops via aqua need to be executed from directory inside repository where config file aqua.yaml exists
+	// Ensuring that sops is executed from directory where .sops.yaml is found permit to use sops via aqua in case of a repository
+	// If .sops.yaml is not found, sops is executed from user home directory like before
+	let sopsConfigUri = await findSopsConfigRecursive(uri.with({ path: path.dirname(uri.path) }));
```

Comment:
I think that what you try to achieve is to run and find the config file .sops.yml in the context of a decrypted file and change CWD to the folder.
The `agua` comments are useful for a description of your intention but in general, it's not related to agua at all.
So please just put the agua-related comments only to the PR description, Maybe to the commit message as an example of usage. But not in js comments. It can be misleading in future.

Reply:
Force push commit [1c64602](https://github.com/signageos/vscode-sops/pull/73/commits/1c64602fc4a21a9c9863a03eb10500d6e2f2d294) less comments related to aqua.